manifests/templates/cdi-controller.yaml.in

```diff
@@ -103,4 +111,212 @@ spec:
     singular: datavolume
   scope: Namespaced
   version: v1alpha1
-
+---
+kind: ServiceAccount
```

Comment:
Any particular reason we can't use the existing cdi-sa Service Account?

Reply:
The api server doesn't need all the permissions cdi-sa has.

Reply:
Sounds reasonable to me.